test/srt/test_disaggregation.py

```diff
@@ -0,0 +1,137 @@
+import subprocess
+import threading
+import time
+import unittest
+from types import SimpleNamespace
+
+import requests
+import torch
+
+from sglang.srt.utils import kill_process_tree
+from sglang.test.few_shot_gsm8k import run_eval as run_eval_few_shot_gsm8k
+from sglang.test.test_utils import (
+    DEFAULT_TIMEOUT_FOR_SERVER_LAUNCH,
+    DEFAULT_URL_FOR_TEST,
+    CustomTestCase,
+    popen_launch_pd_server,
+    run_with_timeout,
+)
+
+
+class TestDisaggregationMooncake(CustomTestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.model = "lmsys/sglang-ci-dsv3-test"
```

Comment:
For faster CI, can we use a small llama model within a node? (maybe tp4+tp4)

Reply:
this test model is lite